engine/common/synchronization/engine.go

```diff
@@ -533,17 +549,16 @@ func (e *Engine) onRangeRequest(originID flow.Identifier, req *messages.RangeReq
 	}
 	err := e.con.Unicast(res, originID)
 	if err != nil {
-		return fmt.Errorf("could not send range response: %w", err)
+		e.log.Warn().Err(err).Msg("sending range response failed")
```

Comment:
Better to log with context here.
```suggestion
		e.log.Warn().Err(err).Hex("origin_id", originID).Msg("sending range response failed")
```